### PR TITLE
feature: add dashboard cta to docs

### PR DESCRIPTION
**Pull request by bencodezen**

### CODE REVIEW

**Comment by amirrustam** [comment]
Posting screenshots here as part of the review. Currently, this is how desktop and mobile look like:

Desktop:
Screen Shot 2020-08-25 at 1 40 05 PM

Mobile:
Screen Shot 2020-08-25 at 1 40 26 PM

File: themes/cypress/layout/partial/dashboard-cta.swig [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<aside class="dashboard-ad">
+  <div class="dashboard-ad-content">
+    <h2 class="dashboard-ad-title">Test and debug faster with the Cypress Dashboard</h2>
+    <ul>
+      <li>See exact point of failure of tests running in CI</li>
```
Comment (jennifer-shehane) [inline themes/cypress/layout/partial/dashboard-cta.swig:5]:
```suggestion
      <li>See the exact point of failure of tests running in CI</li>
```

**Review by jennifer-shehane — CHANGES_REQUESTED** [review]
Agree with Amir that there needs to be some padding above it cause it seems smushed up to the content. 

The screenshot quality is really poor. All of it is fuzzy on my screen. Would like a better quality screenshot.

Screen Shot 2020-08-26 at 2 43 42 PM

**Comment by jordanjwoods** [comment]
@amirrustam @bencodezen what do we need to do to push this through? Do I need to get a high-res image?

**Comment by bencodezen** [comment]
@amirrustam @jennifer-shehane 

I've updated the spacing above the CTA. Good recommendation!

Screen Shot 2020-08-28 at 10 15 27 AM

To account for the screenshot resolution issue, I'm proposing to remove it on mobile at this point so we can get out an iteration of it since it's less noticeable on desktop.

Screen Shot 2020-08-28 at 10 16 38 AM

@jordanjwoods Would love your thoughts on this when you have a moment too!

**Comment by jordanjwoods** [comment]
@bencodezen I support the change! Also, I've reached out to our designer about getting higher res image assets.

**Comment by amirrustam** [comment]
Great, after getting hi-res image and resolving failing tests, this is good to go.

**Comment by jordanjwoods** [comment]
@bencodezen I received a 300ppi and 150ppi version of the assets from our designer. Can you see if these will work? 
150ppi
300ppi

**Pull request merged**